browser/help/faq/technical-details/why-is-this-variant-linked-to-the-wrong-dbsnp-rsid.md

```diff
@@ -4,7 +4,7 @@ question: 'Why is this variant linked to the wrong dbSNP rsID?'
 
 First, we do not continuously update the dbSNP rsIDs displayed in the browser, so there will be some differences between the databases and dbSNP depending on versions being used. The dbSNP version used are:
 
-- gnomAD v4.0: dbSNP version 156
+- gnomAD v4.0 and 4.1: dbSNP version 156
```

Comment:
Do you want to split this to two lines like we do for v3.1 and v3? Just when people search "gnomad v4.1".